app/controllers/api/v1/tags_controller.rb

```diff
@@ -18,6 +18,38 @@ def show
 
         render :json => tag
       end
+
+      # Present these as tags
+      private_class_method def self.api_doc_definition
+        @api_doc_definition ||= api_doc.definitions["Tag"]
+      end
```

Comment:
Where is this used?

Reply:
Was able to remove that, on topology the controller is `Taggings` so they needed to overload the method to return the definition for "Tag", we don't have to do that since we named ours differently.